index.html

```diff
@@ -51,7 +51,8 @@ <h6 class="h6 header-6">Log out</h6>
       <div class="desktop-navbar">
         <a href="./index.html">Skilly</a>
         <div class="page-bar">
-          <h4 class="h4"><a href="./about.html" id="about-h4">About</a></h4>
+          <h4 class="h4"><a href="./index.html" id="home">Home</a></h4>
```

Comment:
Your home page was fine and you did not need to make any changes there. Now I can see that there is no link to the About page on your Home page.
What I mean is:
[ ] Please kindly ensure that there is a link to your home page from your about page and vice versa.